### DISCUSSION TITLE
Propagate Height to `GetValue` recipient

### DISCUSSION THREAD

**Discussion by ancazamfir** [circlefin/malachite#147] [Specifications]
From a discussion with @romac 

We should add the height to `GetValueAndScheduleTimeout(Round, Timeout)`
This will likely be a message to the proposer builder which  probably doesn’t know the height but needs to figure out the slot  based on this height (most likely the slot of previous height) and consensus round.
We should also split this in two outputs:

```
GetValue(Height, Round, Timeout)
ScheduleTimeout(Timeout)
```
The Driver can interact with the host system to start the Propose timer while the `GetValue` message, when received by the proposal builder will give an estimate on the maximum time available for building and propagating a full proposal. 
Note: Another thought was to have the proposal builder initialized with the ProposerTimeout but this increases as we move to new rounds.

@josef-widder @cason 
Thought to bring this up while you are working on https://github.com/informalsystems/malachite/issues/81

**Comment by cason** [comment]
I agree splitting the call. I don't like the format it has now.
